### ISSUE TITLE
Forcing the plugin to load

### ISSUE THREAD

**Issue by Bobakanoosh** [seanbyrne88/RustPlugins#6]
NOTE: There is no way of ensuring load order of the plugins so you may want to rename the .cs file to ensure it gets loaded first. That way subsequent plugins will not throw the warning above because of the load order

This is false in a way. If you're making your own plugin and you use

// Requires: Discord

at the top, then it will require your plugin with that plugin.

**Comment by seanbyrne88** [comment]
That's true but if someone doesn't want to use either Discord or Slack, I don't want them to have to download the plugin.

**Comment by Bobakanoosh** [comment]
If they don't want to use Discord or Slack with their plugin, they shouldn't be putting //Requires: Discord in their plugin anywho?
